api/client/client.go

```diff
@@ -997,3 +997,48 @@ func (c *Client) IsMFARequired(ctx context.Context, req *proto.IsMFARequiredRequ
 	}
 	return resp, nil
 }
+
+// GetToken returns a provision token by name.
+func (c *Client) GetToken(ctx context.Context, name string) (types.ProvisionToken, error) {
+	if name == "" {
+		return nil, trace.BadParameter("missing name")
```

Comment:
Nit: Prefer error messages that provide a bit more context (e.g. `missing token name`, or even `cannot get token, missing name`).